configs/components/_base-rubygem.rb

```diff
@@ -37,6 +37,8 @@
 pkg.url("https://rubygems.org/downloads/#{name}-#{version}.gem")
 pkg.mirror("#{settings[:buildsources_url]}/#{name}-#{version}.gem")
 
-pkg.install do
-  "#{settings[:gem_install]} #{name}-#{version}.gem"
-end
+unless name =~ /nokogiri/
+  pkg.install do
+    "#{settings[:gem_install]} #{name}-#{version}.gem"
+  end
+end
```

Comment:
This is just a placeholder, I think the long term solution is to change this to just add to the install command something like `#{pkg.gem_install_options}`, where we define the pkg.gem_install_options in the nokogiri component.
I can't quite wrap my head around how to implement this yet though.

Reply:
Thanks Josh, took a bit to wrap my head around things but the prior to the instance_eval was the key point. I've pushed up my changes. Working on last little bit now, vanagon-generic-job is failing osx-12-arm, which is cross compiled. 
Thats the last thing I need to resolve.